### PR TITLE
 📝 docs: add Dokploy deployment guide for lobechat-database

### PR DESCRIPTION
**Pull request by SpeedupMaster**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [ ] 👷 build
- [ ] ⚡️ perf
- [x] 📝 docs
- [ ] 🔨 chore

#### 🔀 变更说明 | Description of Change

使用Dokploy部署LobeChat数据库版 #4916

### CODE REVIEW

File: docs/self-hosting/server-database/dokploy.zh-CN.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+# 在 Dokploy 上部署服务端数据库版
+本文将详细介绍如何在 Dokploy 中部署服务端数据库版 LobeChat。
+## 一、准备工作
+### 部署 Dokploy 并进行相关设置
+```shell
+curl -sSL https://dokploy.com/install.sh | sh
+```
+1. 在 Dokploy 的 Settings / Git 处根据提示将 Github 绑定到 Dokploy
+   
+![](https://beta.appflowy.cloud/api/file_storage/e4e5bffb-f416-4f28-a3d8-e4a55237250d/blob/13853032171380694273.png)
```
Comment (arvinxx) [inline docs/self-hosting/server-database/dokploy.zh-CN.mdx]: 所有图片我们都用 github 的地址的，可以传到评论区拿 url

Comment (SpeedupMaster) [reply docs/self-hosting/server-database/dokploy.zh-CN.mdx]: OK，晚点我更新下

**Comment by SpeedupMaster** [comment]
Screenshot 2024-12-10 at 11-15-37 Dokploy

**Comment by SpeedupMaster** [comment]
Screenshot 2024-12-10 at 11-15-55 Dokploy

**Comment by SpeedupMaster** [comment]
Screenshot 2024-12-10 at 11-17-00 Dokploy

**Comment by SpeedupMaster** [comment]
Screenshot 2024-12-10 at 11-17-36 Dokploy

**Comment by SpeedupMaster** [comment]
Screenshot 2024-12-10 at 11-18-04 Dokploy

**Comment by SpeedupMaster** [comment]
Screenshot 2024-12-10 at 11-20-44 Dokploy

**Comment by SpeedupMaster** [comment]
Screenshot 2024-12-10 at 11-21-01 Dokploy

**Comment by SpeedupMaster** [comment]
Screenshot 2024-12-10 at 11-22-09 Dokploy

**Comment by SpeedupMaster** [comment]
Screenshot 2024-12-10 at 11-24-23 Welcome to LobeChat Your personal A  · LobeChat

**Comment by King-WKR** [comment]
> Screenshot 2024-12-10 at 11-20-44 Dokploy

请问步骤里部署的environment是哪个文件里的信息呀

**Comment by SpeedupMaster** [comment]
> > Screenshot 2024-12-10 at 11-20-44 Dokploy
> 
> 请问步骤里部署的environment是哪个文件里的信息呀

建议你看下vercel部署，里面的环境变量介绍的很清楚，dokploy除了最上面4个外，基本上和vercel一样 [ref](https://lobehub.com/zh/docs/self-hosting/server-database/vercel)

**Review by arvinxx — APPROVED** [review]
LGTM~ Thanks for your contribution!

**Pull request merged**